en/01_Optuna_Quick_Start.ipynb

```diff
@@ -0,0 +1,358 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "33Z9NHT6_Eva"
+   },
+   "source": [
+    "<img src=\"files/logo.jpg\"/>\n",
+    "\n",
+    "Optuna is an automatic hyperparameter optimization software framework, particularly designed for machine learning. It features an imperative, define-by-run style user API.\n",
+    "\n",
+    "- GitHub: https://github.com/pfnet/optuna\n",
+    "- Document: https://optuna.readthedocs.io/en/stable/\n",
+    "\n",
+    "This notebook describes the basic usage of Optuna with simple optimization tasks of quadratic function and linear regression."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AEomu83y6MCG"
+   },
+   "source": [
+    "## Installation\n",
+    "First of all, install Optuna by running the following cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "Ow12BEEu6MRQ"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install optuna"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Basic Usage\n",
+    "Below is the basic usage of Optuna. You can immediately start an optimization task just filling the following template with your machine learning logic and the number of trials.\n",
+    "\n",
+    "```python\n",
+    "import optuna\n",
+    "\n",
+    "def objective(trial):\n",
+    "    some_machine_learning_logic()  # Write your machine learning logic here.\n",
+    "    return evaluation_score  # Return the evaluation score of the trained model.\n",
+    "\n",
+    "study = optuna.create_study(direction='minimize')\n",
+    "study.optimize(objective, n_trials=N_TRIALS)  # Specify the number of trials. \n",
+    "```\n",
+    "\n",
+    "## Optimize Quasratic Function\n",
+    "Before optimizing a machine learning model, let's see how Optuna solves a very simple task that minimizes the output of $f(x) = (x - 2)^2$.\n",
+    "Although the answer obiously is $f(x) = 0$ when $x = 2$, Optuna doesn't know how to solve that."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import optuna  # Don't forget installing optuna with `!pip install optuna`.\n",
```

Comment:
I think `forget to install` may be suitable for the situation if this sentence means that please install optuna before you execute this code.